activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Define generators default database configuration
+
+    By default, migrations / models will get generated for the primary database.
+    Users can modify this behaviour in order to point to another database instead.
```

Comment:
I think that this is worded in kind of a confusing way. If no database option is passed AR will assume `db/migrate` for the migrations path, regardless of what is defined in your config. All the migration generator really does is use the db name to find the migration path set on the configuration. I think this should say

```suggestion
    By default, migrations will be generated in the db/migrate directory and models will get generated for the first database defined in the configuration.
    
    Users can modify this behavior to generate those modes with a different default database configuration.
```

Reply:
Done in https://github.com/rails/rails/pull/51161/commits/a1d3b325e4c30d3b36da42be51c67b0bbaa5ee97.